internal/cronsumer.go

```diff
@@ -65,9 +72,11 @@ func (k *kafkaCronsumer) Listen(ctx context.Context, cancelFuncWrapper *func())
 				k.cfg.Logger.Errorf("Error sending next iteration KafkaMessage: %v", err)
 			}
 
+			k.metric.TotalIgnoredMessagesCounter++
```

Comment:
If a message exceeds the retry count, it will be discarded (if the dead letter feature is not enabled); I think this is not correct place for this metric bro